### PR TITLE
Add python 3.10.8 benchmark results

### PR DESCRIPTION
**Pull request by Yuyuutsu**
### What is the context of this PR?
Add benchmark results for updating runner to  Python 3.10.8 

### How to review 
- Have a look at the results and see if they are okay. Got some fails on the baseline results
- Add whether we should approve or disapprove the merge

### CODE REVIEW

**Comment by Yuyuutsu** [comment]
[Upgrade runner to Python 3.10 (2 SP)](https://trello.com/c/33ZeJcSt/5228-upgrade-runner-to-python-310-2-sp)

File: doc/performance-investigations/0021-upgrade-python-to-3.10.8/outputs/baseline/2023-01-03/Results_For_Latest_Benchmark.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+90th: 91ms
+95th: 107ms
+99th: 681ms
+99.9th: 1644ms
+---
+GETs (99th): 638ms
+POSTs (99th): 753ms
+---
+Total Requests: 65,122
+Total Failures: 40
```
Comment (berroar) [inline doc/performance-investigations/0021-upgrade-python-to-3.10.8/outputs/baseline/2023-01-03/Results_For_Latest_Benchmark.yaml:13]: In future prob worth running the baseline first and the test not too long after to ensure the test is consistent. Prob need to investigate these errors to see if they are related to the upgrade and/or maybe running the test again? I don't think I had any errors when I ran my benchmark 🤔

File: doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| Questionnaire GETs  | 638ms    | 468ms         |
+| Questionnaire POSTs | 753ms    | 161ms         |
+| All requests        | 681ms    | 354ms         |
+| Total Requests      | 65,122   | 65,962        |
+| Total Failures      | 40       | 0             |
+| Error Percentage    | 0.06%    | 0.0%          |
+
+## Decision
+
+Merge or discard?
```
Comment (berroar) [inline doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md:33]: Hard to make a decision/view see if there is a performance hit with the current baseline. Ideally the baseline would be similar to our daily test output.

File: doc/performance-investigations/0021-upgrade-python-to-3.10.8/outputs/baseline/2023-01-03/Results_For_Latest_Benchmark.yaml (hunk 1 above)
Comment (Yuyuutsu) [reply doc/performance-investigations/0021-upgrade-python-to-3.10.8/outputs/baseline/2023-01-03/Results_For_Latest_Benchmark.yaml:13]: Yeah seemed odd, didn't get these errors the first time I ran baseline tests. I'll re run the benchmarks for both baseline and investigation

File: doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Results
+
+Results based on 99th percentile timings.
+| Metric              | Baseline | Investigation |
+| ------------------- | -------- | ------------- |
+| Questionnaire GETs  | 469ms    | 433ms         |
+| Questionnaire POSTs | 306ms    | 186ms         |
+| All requests        | 409ms    | 341ms         |
+| Total Requests      | 65,857   | 65,678        |
```
Comment (MebinAbraham) [inline doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md:27]: The numbers don't really make sense. How can Investigation be faster by serve less requests?.

Comment (Yuyuutsu) [reply doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md:27]: I am not sure what you mean , could you explain it to me. I can re run the benchmarks again for both investigation and baseline to see if the results are different 👍

Comment (MebinAbraham) [reply doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md:27]: As in, the numbers suggest that the investigation (3.10) requests are faster, but it ended up serving less total requests which doesn't make sense?. If the requests were faster then, the Total Requests should have been higher right?.

Comment (Yuyuutsu) [reply doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md:27]: Oh I see, that is strange. 
I shall re run the benchmark for a longer duration rather then the default 20 minutes and get more accurate averages.

**Review by MebinAbraham — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Pull request merged**